### PR TITLE
Initial work

### PR DESCRIPTION
**Pull request by AnthonyGiorgio**
Initial port of `sshpass`.

### CODE REVIEW

File: check.sh [hunk 1]
```diff
@@ -0,0 +1,83 @@
+#
```
Comment (IgorTodorovskiIBM) [inline check.sh]: You can probably remove this file

File: cicd.groovy [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+                checkout([
+                        $class: 'GitSCM',
+                        branches: [[name: '*/main']],
+                        doGenerateSubmoduleConfigurations: false,
+                        extensions: [],
+                        userRemoteConfigs: [[url: 'https://github.com/ZOSOpenTools/lessport.git']]])
+        }
+
+        stage('Build') {
+                build job: 'Port-Pipeline', parameters: [string(name: 'PORT_GITHUB_REPO', value: 'https://github.com/ZOSOpenTools/lessport.git'), string(name: 'PORT_DESCRIPTION', value: 'The less terminal pager program.' )]
```
Comment (IgorTodorovskiIBM) [inline cicd.groovy]: This should point to sshpassport rather than lessport

**Review by MikeFultonDev — COMMENTED** [review]
Can you add a short readme that says what it is ?

**Comment by AnthonyGiorgio** [comment]
> Can you add a short readme that says what it is ?

Done.

File: check.sh (hunk 1 above)
Comment (AnthonyGiorgio) [reply check.sh]: Done.

File: cicd.groovy (hunk 2 above)
Comment (AnthonyGiorgio) [reply cicd.groovy]: Fixed.

**Review by IgorTodorovskiIBM — APPROVED** [review]

**Pull request merged**